core/tests/unit/ldml/keyboards/k_003_transform.xml

```diff
@@ -3,55 +3,58 @@
 <!--
 from https://github.com/unicode-org/cldr/blob/keyboard-preview/docs/ldml/tr35-keyboards.md#element-transform
 
-Note: keys should start with [K_A][K_B][K_C] not [K_Y] when transforms are functioning properly.
-
-#TODO-LDML: transforms broken
-#@@keys: [K_Y][K_BKQUOTE][K_A][K_B][K_D][K_BKQUOTE][K_A][K_B][K_E][K_Q][K_BKQUOTE][K_B][K_C]
-#@@expected: y\u0020xd\u0020xeq\u0020m
-
-@@keys: [K_T][K_O][K_D][K_O]
-@@expected: todo
+@@keys: [K_Q][K_U][K_BKQUOTE][K_E]
+@@expected: qu\u00ea
 
 -->
 <!DOCTYPE keyboard SYSTEM "../../../../../resources/standards-data/ldml-keyboards/techpreview/dtd/ldmlKeyboard.dtd">
 <keyboard locale="en" conformsTo="techpreview">
   <info author="srl295" indicator="🙀" layout="qwerty" normalization="NFC" />
 
   <names>
-    <name value="TestKbd" />
+    <name value="TransformKbd" />
   </names>
 
   <keys>
-    <key id="backslash" to="\u{005c}" />
-    <key id="bquote" to="`" />
-    <key id="equals" to="=" />
+    <key id="equal" to="=" />
     <key id="hyphen" to="-" />
+    <key id="untransformed-hat" to="^" transform="no"/>
+    <key id="hat" to="^"/>
   </keys>
 
   <layers form="us">
-    <layer id="base">
-      <row keys="space" /> <!-- number row -->
+    <layer modifier="none" id="base">
+      <row keys="hat 1 2 3 4 5 6 7 8 9 0 hyphen equal" />
       <row keys="q w e r t y u i o p" />
       <row keys="a s d f g h j k l" />
       <row keys="z x c v b n m" />
       <row keys="space" />
     </layer>
+    <layer modifier="shift" id="shift">
+      <row keys="untransformed-hat 1 2 3 4 5 6 7 8 9 0 hyphen equal" />
+      <row keys="Q W E R T Y U I O P" />
+      <row keys="A S D F G H J K L" />
+      <row keys="Z X C V B N M" />
+      <row keys="space" />
+    </layer>
   </layers>
 
+  <variables>
+    <string id="e" value="e" />
+  </variables>
+
   <transforms type="simple">
     <transformGroup>
-      <transform from="ab" to="x" />
-      <transform from="abc" to="y" />
-      <transform from="abef" to="z" />
-      <transform from="bc" to="m" />
-      <transform from="beq" to="n" />
+      <transform from="^${e}" to="ê"/>
     </transformGroup>
   </transforms>
   <transforms type="backspace">
     <transformGroup>
+      <!-- 'ab' deleted as a group -->
       <transform from="ab" />
     </transformGroup>
     <transformGroup>
```

Comment:
Is this reorder group really intended to be under bksp?

Reply:
oops, no.  thanks